crypto/src/openssl.rs

```diff
@@ -42,22 +42,31 @@ impl Hasher for OpensslHasher {
 }
 
 #[cfg(feature = "deterministic_rand")]
-pub struct OpensslCrypto(StdRng);
+pub struct OpensslCrypto {
+    rng: StdRng,
+    // Cached CDI for exported CDI operations.
+    exported_cdi: Option<Vec<u8>>,
```

Comment:
no need to cache

Reply:
I think for the default platform, we should probably just return an actual CDI. It's just for the Caliptra overload that we'll do this extra handle stuff.

Reply:
I needed it for `get_exported_cdi` but with `SignWithExtended` happening in SW instead, I don't think I will need these cached cdis. 

I've removed them.